### PR TITLE
Feature#15/create admin login view

### LINKED ISSUE #15: Create view to login as administrator
# REQUIREMENTS
- Create the index view based on the design 
- Add a the path to the button on employees index to change to admin view 
- Add the path on the admins button to change to the employeevi

# DESIGN
image

### PR DESCRIPTION
**Pull request by nallely64**
# Description
 - What?: Admin´s view implementation and added button´s vinculation with their routes.
 - Why?: To have frontend done and be able to add the backend logic.
 For more information, visit #15 issue.

  ## Screenshots
image (2)

### CODE REVIEW

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <div class="header">
   <h1 class="title">Check In <i class="fa-solid fa-clock"></i></h1>
-  <button class="primary-button-style">
-    Admin Tools
-  </button>
+    <% if current_page?(:controller => 'admin', :action => 'login') %>
```
Comment (alexs) [inline app/views/layouts/_header.html.erb]: According to Ruby 3 guide style, is better use shorthand hash literal syntax when all keys are symbols.

`# bad
{ :a => 1, :b => 2 }

# good
{ a: 1, b: 2 }`

**Review by alexs — DISMISSED** [review]
Good Job 👍

File: app/assets/stylesheets/pages/_admin_login.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+.main_login {
+  padding-top: 5rem;
+
+  h1 {
+    color: $wheat_button;
+    margin-top: 0;
+  }
+
+  form {
+    label{
```
Comment (JuanequeX) [inline app/assets/stylesheets/pages/_admin_login.scss]: Please add space between label and your brace

File: app/views/admin/login.html.erb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    <h1>Admin login</h1>
+    <%= form_with(model: @employee) do |form| %>
+        <div>
+          <%= form.label :email, class: "title" %><br>
+          <%= form.email_field  :email, class: "input-text", placeholder: "admin@hotmail.com"%>
+          <br>
+          <%= form.label :private_number, class: "title" %><br>
+          <%= form.password_field  :private_number, class: "input-text", placeholder: "********"%>
+        </div>
+        <%= form.submit "Sign in", class: "secondary-button-style" %>
```
Comment (JuanequeX) [inline app/views/admin/login.html.erb]: This block have an extra space of indentation, please fix it

File: app/views/layouts/_header.html.erb (hunk 1 above)
Comment (JuanequeX) [reply app/views/layouts/_header.html.erb]: i aggre with @alexs, can read more about the ruby syntax [here](https://www.rubyguides.com/2019/10/ruby-syntax/)

**Review by JuanequeX — DISMISSED** [review]

File: app/views/layouts/_header.html.erb (hunk 1 above)
Comment (germanruzca) [reply app/views/layouts/_header.html.erb]: Fixed, and thanks! @JuanequeX @alexs

File: app/assets/stylesheets/pages/_admin_login.scss (hunk 2 above)
Comment (germanruzca) [reply app/assets/stylesheets/pages/_admin_login.scss]: Done

File: app/views/admin/login.html.erb (hunk 3 above)
Comment (germanruzca) [reply app/views/admin/login.html.erb]: Done

**Comment by germanruzca** [comment]
@alexs Can you check it again? Thanks

**Review by Javi1499 — APPROVED** [review]
Good job

**Review by VictorElizalde — APPROVED** [review]
LGTM

**Review by EliasUR — APPROVED** [review]
LGTM

**Pull request merged**